### ISSUE TITLE
Empty values when reading from a file

### ISSUE THREAD

**Issue by yloiseau** [lucapette/fakedata#67]
When reading values from a file, if the last line ends with `\n` (which is often the case), and empty value is generated.

For instance:
```bash
$ printf 'a\nb\nc' >| a
$ cat -A a
a$
b$
c
$ fakedata -l 10 file:a
a
b
b
a
c
b
a
c
a
a
```

but

```bash
$ printf 'a\nb\nc\n' >| a
$ cat -A a
a$
b$
c$
$ fakedata -l 10 file:a
a
b

a

b
c
a

c
```

The file can't therefore be generated using another command. For instance

```bash
$ find . -name '*.txt' > textfiles.dat
$ fakedata -l 10 file:textfiles.dat
```

since every such command (or even editor), and `fakedata` itself, add a `\n` on the last line.

**Comment by kevingimbel** [comment]
I added some code to remove new lines and empty values but maybe that's a bit too much. 

I'm not sure yet if it is a use-case but maybe someone wants empty values? The new code would remove all new lines and empty values. https://github.com/lucapette/fakedata/blob/588edff8e595b8f98aaffac3eaa4965e820bac8c/pkg/fakedata/generator.go#L191-L195 

I'll sleep on this.

_ping @lucapette for discussion_ :)

**Comment by lucapette** [comment]
@yloiseau thank you for the fantastic report! @kevingimbel thank you so much for being active on the project despite how busy you're at the moment. I took some time this morning and pushed a #68 which, based on my understanding, fixes the problem presented here.

I also updated one of our integration tests to expose the fix, so I'm kind of confident this works. If you're able to test the PR, it would be great. Otherwise later today or early next week, I'll do some more testing before cutting a new release
